verticapy/tests_new/machine_learning/vertica/linear_model/test_linear_regression.py

```diff
@@ -0,0 +1,574 @@
+"""
+(c)  Copyright  [2018-2023]  OpenText  or one of its
+affiliates.  Licensed  under  the   Apache  License,
+Version 2.0 (the  "License"); You  may  not use this
+file except in compliance with the License.
+
+You may obtain a copy of the License at:
+http://www.apache.org/licenses/LICENSE-2.0
+
+Unless  required  by applicable  law or  agreed to in
+writing, software  distributed  under the  License is
+distributed on an  "AS IS" BASIS,  WITHOUT WARRANTIES
+OR CONDITIONS OF ANY KIND, either express or implied.
+See the  License for the specific  language governing
+permissions and limitations under the License.
+"""
+import math
+import verticapy.machine_learning.vertica as vpy_linear_model
+from verticapy.connection import current_cursor
+import numpy as np
+import sklearn.metrics as skl_metrics
+import sklearn.linear_model as skl_linear_model
+import pytest
+import statsmodels.api as sm
+from scipy import stats
+import matplotlib.pyplot as plt
+
+REL_TOLERANCE = 1e-2
```

Comment:
Shouldn't the results be closer than just 1e-2? I can understand in certain cases the calculation method may differ but then we can tweak the tolerance for those specific cases. It will be good to know which attributes are a bit off.